### PR TITLE
Add separate sensors for power and current

### PR DESCRIPTION
**Pull request by krconv**
Moves the power sensors to be nested under the CT clamps; mainly so that when we figure out how the current sensors work, we won't need to make a breaking change to config

### CODE REVIEW

File: esphome/components/emporia_vue/emporia_vue.h [hunk 1]
```diff
@@ -42,16 +42,16 @@ struct __attribute__((__packed__)) SensorReading {
...
 
 class EmporiaVueComponent : public Component, public i2c::I2CDevice {
  public:
   void dump_config() override;
 
   void set_sensor_poll_interval(uint32_t sensor_poll_interval) { this->sensor_poll_interval_ = sensor_poll_interval; }
   uint32_t get_sensor_poll_interval() const { return this->sensor_poll_interval_; }
-  void set_phases(std::vector<PhaseConfig *> phases) { this->phases_ = std::move(phases); }
-  void set_ct_sensors(std::vector<CTSensor *> sensors) { this->ct_sensors_ = std::move(sensors); }
+  void set_phases(std::vector<PhaseConfig *> phases) { this->phases_ = phases; }
```
Comment (Maelstrom96) [inline esphome/components/emporia_vue/emporia_vue.h]:
```suggestion
  void set_phases(std::vector<PhaseConfig *> phases) { this->phases_ = std::move(phases); }
```
Can you please reverse this.

File: esphome/components/emporia_vue/emporia_vue.h [hunk 2]
```diff
@@ -42,16 +42,16 @@ struct __attribute__((__packed__)) SensorReading {
...
+  void set_ct_clamps(std::vector<CTClampConfig *> ct_clamps) { this->ct_clamps_ = ct_clamps; }
```
Comment (Maelstrom96) [inline esphome/components/emporia_vue/emporia_vue.h]:
```suggestion
  void set_ct_clamps(std::vector<CTClampConfig *> ct_clamps) { this->ct_clamps_ = std::move(ct_clamps); }
```

**Review by Maelstrom96 — CHANGES_REQUESTED** [review]
Looks pretty good other than the lint issues - Will need to do the changes in esphome-docs.

**Comment by krconv** [comment]
I fixed the linting issues in https://github.com/Maelstrom96/esphome/pull/6, sorry for the noise; closing this one

**Pull request merged**